Buildings/Controls/OBC/FDE/DOAS/ExhaustFanController.mo

```diff
@@ -0,0 +1,153 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block ExhaustFanController
+  "This block manages start, stop, and speed of the exhaust fan."
+
+  parameter Real bldgSPset(
+   final unit="Pa",
+   final quantity="PressureDifference")=15
+    "Building static pressure set point";
+
+    // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput supFanProof
+    "True when supply fan is proven on"
+      annotation (Placement(transformation(extent={{-142,34},{-102,74}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput bldgSP
+    "Building static pressure"
+      annotation (Placement(transformation(extent={{-142,-28},{-102,12}}),
+        iconTransformation(extent={{-140,-84},{-100,-44}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput exhFanProof
+    "True when exhaust fan is proven on."
+      annotation (Placement(transformation(extent={{-142,-62},{-102,-22}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+
+     // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput exhFanStart
```

Comment:
May not need this output as it is basically the input `supFanProof`. You may directly use the input for enabling fan.

Reply:
This method is preferred to make the logic easier to follow in the final combined controller.